packages/lab/stories/static-list/static-list.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import type { Meta, StoryFn } from "@storybook/react";
+import React, { Fragment, useState } from "react";
+import { complexEventsData, eventsData } from "../assets/exampleData";
+
+export default {
+  title: "Lab/Static List",
+  component: StaticList,
+} as Meta<typeof StaticList>;
+
+export const Default: StoryFn<StaticListProps> = () => {
+  const [listCount, setListCount] = useState(3);
+
+  const handleListItem = () => {
+    setListCount((prev) => prev + 1);
+  };
+  const handleReset = () => {
+    setListCount(3);
+  };
+  return (
+    <StackLayout>
+      <StackLayout direction={"row"}>
+        <Button
+          onClick={handleListItem}
+          disabled={listCount > eventsData.length - 1}
+        >
+          {`Show next ${listCount + 1} events`}
+        </Button>
+        <Button onClick={handleReset}>Reset</Button>
+      </StackLayout>
+      <StaticList style={{ width: "320px", height: "250px" }}>
+        {eventsData.slice(0, listCount).map((event, _index) => (
+          <StaticListItem key={_index}>
+            <StaticListItemContent>
+              <Text>{event}</Text>
+            </StaticListItemContent>
+          </StaticListItem>
+        ))}
+      </StaticList>
+    </StackLayout>
+  );
+};
+
+export const ComplexLabel: StoryFn<StaticListProps> = () => {
+  return (
+    <StaticList style={{ width: "320px" }}>
+      {complexEventsData.map(({ title, time }) => (
+        <StaticListItem key={title}>
+          <StaticListItemContent>
+            <StackLayout gap={0.5}>
+              <Text color="inherit">{title}</Text>
+              <Text styleAs="label" color="secondary">
+                {time}
+              </Text>
+            </StackLayout>
+          </StaticListItemContent>
+        </StaticListItem>
+      ))}
+    </StaticList>
+  );
+};
+
+export const WithIcons: StoryFn<StaticListProps> = () => {
+  return (
+    <StaticList style={{ width: "320px" }}>
+      {complexEventsData.map(({ title, time }) => (
+        <StaticListItem key={title}>
+          <CalendarIcon />
+          <StaticListItemContent>
+            <StackLayout gap={0.5}>
+              <Text color="inherit">{title}</Text>
+              <Text styleAs="label" color="secondary">
+                {time}
+              </Text>
+            </StackLayout>
+          </StaticListItemContent>
+        </StaticListItem>
+      ))}
+    </StaticList>
+  );
+};
+
+export const WithButtons: StoryFn<StaticListProps> = () => {
+  return (
+    <StaticList style={{ width: "320px" }}>
+      {complexEventsData.map(({ title, time }) => (
+        <StaticListItem key={title}>
+          <StaticListItemContent>
+            <StackLayout gap={0.5}>
+              <Text color="inherit">{title}</Text>
+              <Text styleAs="label" color="secondary">
+                {time}
+              </Text>
+            </StackLayout>
+          </StaticListItemContent>
+          <Button variant="secondary" aria-label="open in another tab">
```

Comment:
I don't think this is labelled correctly

Reply:
it will also need updating in split layout stories, will update if needed once i check with Jake

Reply:
@jake-costa